starters/next-react-query-tailwind/.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+cd "$(dirname -- "$0")" && npm test
```

Comment:
These husky scripts are more focused on the generated kits and not the kit in this context fwiw. If this works in a generated kit, that's great though.